src/CONST.ts

```diff
@@ -930,6 +930,7 @@ const CONST = {
         RECEIPTS: 'receipts@expensify.com',
         STUDENT_AMBASSADOR: 'studentambassadors@expensify.com',
         SVFG: 'svfg@expensify.com',
+        EXPENSIFY_EMAIL_DOMAIN: '@expensify',
```

Comment:
```suggestion
        EXPENSIFY_EMAIL_DOMAIN: '@expensify.com',
```

If you have an phone number as your primary contact method the `userEmail` gets format : `<phone_number>@expensify.sms` so we need to specify top-level domain as well.